cogs/filters.py

```diff
@@ -64,6 +64,45 @@ async def bulk_load_config(self, ctx):
         else:
             await ctx.send("There is no valid file for loading!")
 
+    @is_staff("Helper")
+    @commands.group()
+    async def invitefilter(self, ctx):
+        """Command group for managing the invite filter"""
+        if ctx.invoked_subcommand is None:
+            await ctx.send_help(ctx.command)
+
+    @is_staff("SuperOP")
+    @invitefilter.command(name='add')
+    async def add_invite(self, ctx, invite: discord.Invite, alias: str):
+        """Adds a invite to the filter whitelist"""
+        if await self.bot.invitefilter.fetch(code=invite.code, alias=alias, separator='OR'):
+            return await ctx.send("This invite code or alias is already in use!")
+        name, _ = await self.bot.invitefilter.add(name=invite.guild.name, code=invite.code, alias=alias)
```

Comment:
Won't this fail, due to `uses` in `InviteFilterManager.add` being a required argument afaict?

Reply:
Fixed in d5fb46d